README.md

```diff
@@ -86,6 +86,17 @@ client.autocomplete('custom_fields.brand', 'a', callback);
 client.setAutocompleteSize(20);
 ```
 
+#### Search operator
+Description for search operator. There are three options:
+- **"and"**:
+- **"or"**:
+- **no parameter**: Remove the query parameter from the search api
```

Comment:
Could you elaborate what removing the query parameter means. E.g. behavior defined by backend heuristics?

Reply:
It is to remove "&defaultOperator=or" from the Search API query. Basically the results would be the same as having "&defaultOperator=or" (in case "or" is a default value) but I think we are letting the developers to add a query parameter and we should let them remove it if they want to.

Reply:
Let's not merge this yet, I asked @artem to check the README.md and update the description.